gerador-piadas/src/main/java/br/com/codenation/PiadaServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package br.com.codenation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PiadaServiceImpl implements PiadasService{
```

Comment:
Um outro problema é a nomeclatura `Impl`, Qual é a necessidade?
ele implementa uma interface, pq precisa de um `Impl`, seria semelhante a ter um campo nome com `nomeString`.

* https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c
* https://dzone.com/articles/naming-conventions-from-uncle-bobs-clean-code-phil